parts/kubernetesmastercustomscript.sh

```diff
@@ -169,12 +169,6 @@ function configNetworkPolicy() {
     fi
 }
 
-function ensureEtcd() {
```

Comment:
I have verified by creating a file at the end of a successful etcd setup and then reading it during CSE run. Synchronization is not an issue here since waagent logs still point to the same issue and seem to fail before: https://github.com/Azure/WALinuxAgent/issues/651. 

In the meantime I think this would be a good check to add anyways to CSE run to make sure etcd is up so I am adding it and will update this PR.